pyproject.toml

```diff
@@ -13,7 +13,7 @@ rpaframework-google = "^7.0.0"
 rpaframework-openai = "^1.0.1"
 rpaframework-recognition = "^5.0.1"
 rpaframework-windows = "^7.0.3"
-rpaframework-assistant = "^1.0.4"
+rpaframework-assistant = "^1.0.5"
```

Comment:
Better to run a `poetry update -vvv` (or `inv install.update`) directly in _master_ (alongside this change) once you merge and release the new assistant package, so you get it in the docs straight from PyPI.

(as PyPI releases usually work from _master_ post-merge)

Reply:
we have been building the pypi releases by hand for rpaframework-assistant, but valid point about the workflow.

Reply:
makes sense, I'll remove this change from the PR and we can do it afterwards then

Reply:
```suggestion
rpaframework-assistant = "^1.0.4"
```